stories/src/app/components/DisplayStory.tsx

```diff
@@ -0,0 +1,43 @@
+import StoryPost from "./StoryPost";
+import Form from "next/form";
+
+interface Props {
+    name: string,
+    description: string
+}
+
+export default function DisplayStory(props: Props) {
+    const posts = [
+        {
+            content: "Once upon a time there was a mushroom named Greg. Greg's life ambition was to one day cut every single blade of grass on his lawn to the exact same height down to the millimeter. Every day, he practiced his measuring skills and worked out his scissor muscles so he could perfect this craft.",
+            author: "Hailey",
+            date: new Date("11/19/24")
+        },
+        {
+            content: "Unfortunately for Greg, his neighbor, Persephone, also had exactly one life ambition. Her life ambition was to make Greg sad. So every night, when Greg was asleep, she went out to his lawn and cut exactly one blade lower than all of the others. Sometimes, she even swapped out his scissors with slightly bigger scissors so he would think his finger muscles had gotten smaller.",
```

Comment:
What did Greg ever do to Persephone???

Reply:
Wow attacking Greg for this huh all the man ever wanted was evenly cut grass